### DISCUSSION TITLE
Connection issues

### DISCUSSION THREAD

**Discussion by Mtazione** [Mygod/VPNHotspot#351] [Q&A]
When I'm sharing my connection with one person, it works fine, but when I share it with two or more people, my phone's internet stops working, both for me and for the others. This happens all the time, how can I fix it? I use a S20FE with Magisk and my VPN is bypassed by V2rayNG (own server)

**Comment by Mygod** [comment]
Does it work if you share your Internet using system tethering without using this app?

**Reply by Mtazione** [reply]
Yep, without issues. I don't know what's happening, i'll try other VPN client or something like that.

**Reply by Mygod** [reply]
Does the Internet come back if you turn off VPN sharing (but keep tethering on)?

**Reply by Mtazione** [reply]
Yes, I believe the problem really won't be with VPN Hotspot, there is some sort of problem beyond that.
